style.css

```diff
@@ -1015,19 +1015,45 @@ body:not(:-moz-handler-blocked) #single #content > p:first-child:first-letter {
 
 .related-stories .span3:nth-child(4n+1) { margin-left: 0; }
 
+/* WordPress styles */
+img.alignleft {
+  float: left;
+  margin: 0 12px 12px 0;
+}
+
+img.alignright {
+  float: right;
+  margin: 0 0 12px 12px;
+}
+
 /* Featured Story Styles */
-.featured-story-header { background: #F1F1F1; padding: 1em 20px; }
+.featured-article > .container:first-of-type p:first-of-type:first-letter {
+  font-family: Georgia, serif;
+  color: #FFC907;
+  display: block;
+  float: left;
+  font-size: 48px;
+  font-weight: normal;
+  height: 48px;
+  line-height: 48px;
+  margin-right: 7px;
+  margin-top: 0;
+}
+.featured-story-header { background: #F1F1F1; margin-bottom: 3em; }
```

Comment:
I think removing horizontal padding on .featured-story-header here ended up causing a lack of gutters at the -xs breakpoint--consider adding padding adjustments in a media query specifically for the -xs breakpoint to re-add them.
![screen shot 2018-01-16 at 3 32 38 pm](https://user-images.githubusercontent.com/1649216/35013292-8798e340-fad2-11e7-93f9-560fcce2f92f.png)